providers/apache/spark/tests/system/apache/spark/example_pyspark.py

```diff
@@ -43,12 +42,12 @@ def example_pyspark():
 
     # [START task_pyspark]
     @task.pyspark(conn_id="spark-local")
-    def spark_task(spark: SparkSession, sc: SparkContext) -> pd.DataFrame:
+    def spark_task(spark: SparkSession) -> pd.DataFrame:
         df = spark.createDataFrame(
             [
                 (1, "John Doe", 21),
                 (2, "Jane Doe", 22),
-                (3, "Joe Bloggs", 23),
+                (3, "Joe Blogs", 23),
```

Comment:
It is actually Bloggs though
https://en.wiktionary.org/wiki/Joe_Bloggs#:~:text=English-,Proper%20noun,unknown%20or%20anonymous%20male%20person.